admin/src/templates/upgrade.html

```diff
@@ -93,10 +87,7 @@
         <p ng-hide="versions.branches.length" translate>instance.upgrade.no_branches</p>
         <ul ng-show="versions.branches.length" class="table table-striped">
           <li ng-repeat="release in versions.branches" class="row">
-            <div class="col-xs-5"><span class="label label-info">{{release.id}}</span></div>
-            <div class="col-xs-5">{{release.value.time | date:'medium'}}</div>
```

Comment:
Is it important that this one had `date:'medium'`?

Reply:
Important is a strong term, but yes, we should probably not erase that. Good catch!